go.mod

```diff
@@ -1,72 +1,84 @@
-module github.com/openshift-kni/oran-o2ims
+module oran-o2ims
```

Comment:
Why is this needed?

Reply:
I suspect a go.mod was generated by kubebuilder when I did the initial operator scaffolding. I will change everything back to include github.com/openshift-kni.